portfolio/src/main/webapp/script.js

```diff
@@ -187,3 +187,67 @@ function loadPage(pageNumber) {
   }
   page = pageNumber;  
 }
+
+function checkLogin() {
+  fetch('/login').then(response => response.json()).then(status => {
+    console.log(status);
```

Comment:
l think if this is the last step, we shouldn't console.log() anything, but if it's intermediary, it should be fine.